packages/flutter/test/material/tabs_test.dart

```diff
@@ -859,20 +859,6 @@ void main() {
     expect(tabController.indexIsChanging, false);
   });
 
-  testWidgets('TabBarView child disposed during animation', (WidgetTester tester) async {
-    // This is a regression test for the scenario brought up here
-    // https://github.com/flutter/flutter/pull/7387#discussion_r95089191x
-
-    final List<String> tabs = <String>['LEFT', 'RIGHT'];
-    await tester.pumpWidget(buildLeftRightApp(tabs: tabs, value: 'LEFT'));
-
-    // Fling to the left, switch from the 'LEFT' tab to the 'RIGHT'
-    final Offset flingStart = tester.getCenter(find.text('LEFT CHILD'));
-    await tester.flingFrom(flingStart, const Offset(-200.0, 0.0), 10000.0);
-    await tester.pump();
-    await tester.pump(const Duration(seconds: 1)); // finish the scroll animation
-  });
-
```

Comment:
I removed this test because there is another one below with the same description and because it seems to be a leftover (no call to `expect`).
https://github.com/flutter/flutter/blob/de8f3727cad1b809ce3bb63dae0bce49705337e0/packages/flutter/test/material/tabs_test.dart#L1523